libs/test-setup/src/connectors/mssql.rs

```diff
@@ -39,6 +56,7 @@ pub async fn reset_schema(conn: &dyn Queryable, schema_name: &str) -> Result<(),
         schema_name
     );
 
+    conn.raw_cmd(&drop_views).await?;
     conn.raw_cmd(&drop_fks).await?;
     conn.raw_cmd(&drop_tables).await?;
```

Comment:
Random thought: would concatenating these queries and sending them all at the same time work?

Reply:
No. It will not work. I tried, but the SQL Server told me to try again.